cms/static/js/views/settings/main.js

```diff
@@ -367,12 +367,19 @@ var DetailsView = ValidatingView.extend({
             message = this.post_enrollment_email_elem.val();
         }
 
-        $.post("/settings/send_test_enrollment_email",
+        // Validation for keyword substitution
```

Comment:
Nit: This comment seems superfluous ("The `KeywordValidator` is for 'Validation for keyword...'").